stests/core/orchestration/context.py

```diff
@@ -48,6 +48,9 @@ class ExecutionContext:
     # Label to disambiguate a step within the context of a phase.
     step_label: typing.Optional[str]
 
+    # Flag to indicate use of so-called called contracts, i.e. on chain contracts.
+    use_called_contracts: bool
```

Comment:
I think `use_called_contracts` should be called `use_stored_contracts` to be consistent with our terminology. We can deploy a smart contract in WASM format, the code of the deployed contract will be executed and during the execution it can store a contract (a function that is part of the deployed WASM module). That stored contract can be later called via its hash, name (named key) or uref.

Reply:
OK I was confused on this one - am reverting to use_stored_contracts.